ecs-agent/netlib/model/ecscni/client.go

```diff
@@ -48,9 +50,10 @@ func NewCNIClient(paths []string) CNI {
 func (c *cniClient) Add(ctx context.Context, config PluginConfig) (types.Result, error) {
 	rt := BuildRuntimeConfig(config)
 	net, err := BuildNetworkConfig(config)
-	if err != nil {
+	if err != nil || net == nil || net.Network == nil {
```

Comment:
If the net is nil, will the err be nil or something else?

Reply:
Thanks, err would be nil, so that's not helpful at all. I'll update this.